development/php/tests.Dockerfile

```
@@ -0,0 +1,8 @@
+FROM php:8.4-cli-bookworm
+
+COPY --from=composer/composer:2.9-bin /composer /usr/bin/composer
+
+RUN apt update --quiet --assume-yes \
```

Hadolint (2.14.0)

[info] Delete the apt lists (/var/lib/apt/lists) after installing something

(DL3009)